### ISSUE TITLE
chore: Update build to support python 3.11

### ISSUE THREAD

**Issue by disaac** [lardbit/nefarious#296]
* Python 3.11 has deprecated the import statement used in the celery 4.4.7 installed vine five.py resulting in the following error. 
```
five.py", line 364, in <module>
    from inspect import formatargspec, getargspec as _getargspec  # noqa
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ImportError: cannot import name 'formatargspec' from 'inspect' 
```
* To work around this issue during development ensure you are using python 3.9. 
* See [celery/celery PR-7879](https://github.com/celery/celery/pull/7879)

**Comment by lardbit** [comment]
Yeah, we should upgrade to python 3.11.  nefarious is currently built using python 3.9 (see the Dockerfile) which its EOL is *2025-10*.  So we do have some time.
